wake-lock/idlharness.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Wake Lock API IDL tests</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+<script>
+"use strict";
+
+function doTest([dom_idl, wakelock_idl, wakelock]) {
+  const idl_array = new IdlArray();
+  idl_array.add_untested_idls(dom_idl);
+  idl_array.add_untested_idls('interface Navigator {};');
+  idl_array.add_idls(wakelock_idl);
+  self.wakelock = wakelock;
```

Comment:
Using "self" here is confusing (because self is window). We should just use "window" (see below)

Reply:
Thanks, I will fix it.